docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -106,17 +106,16 @@ kubeadm on, and run:
 kubeadm init
 ```
 
+**Note:** You need to choose a Pod Network Plugin in the next step. Depending on what
```

Comment:
Similar to below, combine these notes by making these bullet points under a single **Note:** label.